csuntechlab/iSTART
# STR-168 -- FE: Fix tab Accessibility on Dashboard and main navigation

# Description

Fixed the accessibility on the main navigation
  
# Testing

1. Login as cyndee.bellamy or april.feldman.
2. Hover over the links in the navbar ad check if there is a border below the words
3. Tab through the navbar and check if the border is below the words
